lib/debug/deprecate.js

```diff
@@ -34,6 +35,8 @@ const assert = require('./assert');
  * - `url` A URL that refers to additional information about the deprecation
  */
 function deprecate(description, condition, options) {
+  const currentEmbeCLIVersion = process.env.OVERRIDE_DEPRECATION_VERSION ?? emberCLIVersion;
```

Comment:
```suggestion
  const currentEmberCLIVersion = process.env.OVERRIDE_DEPRECATION_VERSION ?? emberCLIVersion;
```

I would also declare this above `isDeprecationRemoved`.

Reply:
Ok, I put it above the `isDeprecationRemoved` declaration.